source/GlassView.Export/Exporter.cs

```diff
@@ -0,0 +1,14 @@
+using BenchmarkDotNet.Reports;
+
+namespace GlassView.Export;
+
+public sealed class Exporter(DirectoryInfo path) : IExporter
+{
+    public async Task Export(Summary summary, CancellationToken token)
```

Comment:
Could you maybe rename the summary to "InputSummarry"? That will show that this is our "input" 😊

Reply:
public static class GlassView
{
    public static IExporter Configure(AppConfig config)
    {
 
    }
}

Reply:
Done all, incl app settings.